src/aui-form-validator/js/aui-form-validator.js

```diff
@@ -734,12 +734,12 @@ var FormValidator = A.Component.create({
          * Gets the stack error container of a field.
          *
          * @method getFieldStackErrorContainer
-         * @param {Node} field
+         * @param {Node|String} field
          * @return {Node}
          */
         getFieldStackErrorContainer: function(field) {
             var instance = this,
-                name = field.get('name'),
+                name = isNode(field) ? field.get('name') : field,
                 stackContainers = instance._stackErrorContainers;
```

Comment:
I've thought about doing this across all uses of `field`, to always return the "correct" one. But declined to do it here, for this ticket.

If we want to generally improve the code to support that, I figured we could do it on a dedicated ticket.